### ISSUE TITLE
This repo isn't updated as per the play store version!

### ISSUE THREAD

**Issue by dark-N00B** [nic-delhi/AarogyaSetu_Android#542]
The last commit was way before in 2020 and the playstore version has been updated numerous ever since! Now we're in 2021! Seems like this repo was just created to satisfy some curious minds about security concerns but was never the actual code!

**Comment by sriharisahu** [comment]
You are right. When this repo was created even then we knew that this is not the real code. Hogwash is what the government is good at and that's what this is.
